app/views/judgmentUpload.scala.html

```diff
@@ -0,0 +1,139 @@
+@import java.util.UUID
```

Comment:
This is what I was thinking but it's a bit awkward (especially the form, which makes a call to a template). The "<noscript>" part at the top could be reused since it's exactly the same on both pages but the other parts have context-specific text; it would require a lot of if-else statements and possibly using the "messages" file.

Alternatively, I could pass a case class of strings specific to each page from the upload controller and populate the page; this would probably allow me to make the "upload in progress" section into a template and possibly the second half of the form. What do you think?

Reply:
How many strings do you think you would need to pass in via the case class. If its a lot I don't think its worth doing.

Reply:
1. To move the `<noscript>`, 0
2. To Move the logged-out-error, 0
3. to move the title, instruction and upload-error, 3
4. To move the bottom half of the form (from the dropzone to the start upload buttons, about 6
5. In order to move the "upload in progress" section, should be about 4

Out of the ones that require strings to be passed in (3, 4 and 5), I think 5 would have the biggest significance, since it's large. What do you think?

Also, since 1 and 2 could be moved easily, should I just move them?

Reply:
Go with 1 and 2. Leave the rest for now. We can come back to it, if the maintenance starts to become an issue in the future.

Reply:
Cool. Thank you!